ts/src/sourcemapper/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
This file is not under a third party directory, why is there a separate license file?

Reply:
I moved this to third_party, and kept the license. 
I'm not totally sure what the right approach for adding this file is.